README.md

```diff
@@ -481,7 +481,23 @@ To expose multiple groups you can pass an array of group names:
 A [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) (CSP) may block inline scripts, including those output by Ziggy's `@routes` Blade directive. If you have a CSP and are using a nonce to flag safe inline scripts, you can pass the nonce as as the second argument to the `@routes` directive and it will be added to Ziggy's script tag:
 
 ```php
-@routes(false, 'your-nonce-here')
+// PHP ^8.0
+@routes(nonce: 'your-nonce-here')
+
+// PHP <=7.4
+@routes(null, 'your-nonce-here')
+```
+
+#### Using `@routes` with a custom URL
+
+You can also pass a custom base URL to the `@routes` Blade directive to override it regardless of the current request URL.
+
+```php
+// PHP ^8.0
+@routes(url: 'https://your-url-here.com')
+
+// PHP <=7.4
+@routes(null, null, 'https://your-url-here.com')
```

Comment:
The URLs here should be `https://example.com` which is [reserved for documentation](https://datatracker.ietf.org/doc/html/rfc2606).